src/language-server/simple-ui.langium

```diff
@@ -116,6 +116,7 @@ Type returns string:
 
 hidden terminal WS: /\s+/;
 terminal ID: /[_a-zA-Z][\w_]*/;
+terminal HEADINGLEVEL returns number: /[1-6]/;
```

Comment:
The problem with this approach is that it is now hiding the terminal rule INT. If we have to use INT at some point, it will throw an error if the value is between 1 and 6.

One way we could go around this is to put the check inside of the validator.

Reply:
Yea we need to do this using validation then because moving the terminal rule will break it.